Lab2.ipynb

```diff
@@ -0,0 +1,677 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "### Problem 1\n",
+    "# Part 1A\n",
+    "x0 = np.linspace(0, 10, 11)\n",
+    "dist = stats.poisson.pmf(x0, 5) # mu = 5\n",
+    "\n",
+    "conv1 = np.convolve(dist, dist)\n",
+    "x1 = np.linspace(0, len(conv1), len(conv1))\n",
```

Comment:
`np.linspace` includes both endpoints, so asking for a space between 0 and `len(conv1)` with `len(conv1)` steps will give a list of non-integer values. Instead you should use `np.linspace(0, len(conv1)-1, len(conv1))`. Or even better, use `np.arange(0, len(conv1))` instead of `linspace`. When you want a list of integers `arange` lets you avoid off-by one errors like this completely.